transformation/src/test/test-files/inputs/AntibodyLot/part1.json

```diff
@@ -0,0 +1,4 @@
+{"@id":"/antibodies/ENCAB140BWE/","@type":["AntibodyLot","Item"],"accession":"ENCAB140BWE","aliases":["bradley-bernstein:PchAb 1360"],"alternate_accessions":[],"audit":{"INTERNAL_ACTION":[{"detail":"Antibody {ENCAB140BWE|/antibodies/ENCAB140BWE/} does not have AR dbxrefs.","category":"missing antibody registry reference","level_name":"INTERNAL_ACTION","level":30,"path":"/antibodies/ENCAB140BWE/","name":"audit_antibody_dbxrefs_ar"}]},"award":"/awards/UM1HG009390/","characterizations":["/antibody-characterizations/7c13630f-d583-4979-b604-33f401842949/","/antibody-characterizations/5ede8c39-68d4-4d7d-b121-d94e98bb9a28/"],"clonality":"monoclonal","date_created":"2017-12-06T21:08:41.675624+00:00","dbxrefs":[],"host_organism":"/organisms/rabbit/","lab":"/labs/bradley-bernstein/","lot_id":"9","lot_id_alias":[],"lot_reviews":[{"detail":"Fully characterized.","status":"characterized to standards","biosample_term_name":"any cell type or tissue","targets":["/targets/H3K4me3-human/"],"organisms":["/organisms/human/"],"biosample_term_id":"NTR:99999999"}],"product_id":"9751BC","purifications":[],"schema_version":"9","source":"/sources/cell-signaling/","status":"released","submitted_by":"/users/f1843c60-e027-4b18-8582-64d3f3eae45b/","targets":["/targets/H3K4me3-human/"],"title":"ENCAB140BWE","url":"http://www.cellsignal.com/products/9751.html","used_by_biosample_characterizations":[],"uuid":"46aeb66a-d983-4b6f-ae1c-6aaa86960eff"}
```

Comment:
This line has an audit value, should we add columns to this table to capture it? Ok to leave for a separate ticket (I've got a generic method for summarizing audits on my file branch).

Reply:
Note: We talked about this over slack and decided not to store the audit values for antibodies.